### PR TITLE
Feature/news pods

### PR DESCRIPTION
**Pull request by bh412**

### CODE REVIEW

File: app/js/app/directives/react_classes/PodBlock.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                newDoc.image = newVal;
+
+                this.onDocChange(this, oldDoc, newDoc);
+            },
+
+            render: function() {
+                return (
+                    <Block type="pod" blockTypeTitle="Pod" doc={this.props.doc} onChange={this.onDocChange}>
+                    <form>
+                    <div className="row">
```
Comment (jsharkey13) [inline app/js/app/directives/react_classes/PodBlock.js:51]: Are you missing a title here for physics pods?

File: app/js/app/directives/react_classes/MetaData.js [hunk 2]
```diff
@@ -371,8 +371,8 @@ define(["react", "jquery"], function(React,$) {
 					</div>;
 				}
 
-				if (this.props.doc.type == "emailTemplate") {
-					var emailTemplateMeta = [
+				if (this.props.doc.type == "emailTemplate" || this.props.doc.type == "isaacPod") {
+					var published = [
```
Comment (jsharkey13) [inline app/js/app/directives/react_classes/MetaData.js:375]: What is this change for?

Comment (bh412) [reply app/js/app/directives/react_classes/MetaData.js:375]: It only contains the published option, and we're now using it for pods as well as emails, so I made the name a bit more generic

File: app/js/app/directives/react_classes/PodBlock.js (hunk 1 above)
Comment (bh412) [reply app/js/app/directives/react_classes/PodBlock.js:51]: This was one of the 'I'll explain in a doc and keep it as we have now' - the page title is still there, it's in the metadata like other pages. I'd be happy to put it in as a field, it just would be duplicated

**Comment by bh412** [comment]
Change to tag behaviour implemented

**Pull request merged**